app/models/group_offer.rb

```diff
@@ -0,0 +1,41 @@
+class GroupOffer < ApplicationRecord
+  belongs_to :department, optional: true
+
+  validates :necessary_volunteers, numericality: { greater_than: 0 }, allow_nil: true
+
+  def self.target_collection
+    [:women, :men, :children, :teenagers, :unaccompanied, :all]
+  end
+
+  def self.duration_collection
+    [:long_term, :regular, :short_term]
+  end
+
+  INTERNAL_OFFER = 'internal_offer'.freeze
+  EXTERNAL_OFFER = 'external_offer'.freeze
+  OFFER_STATES = [INTERNAL_OFFER, EXTERNAL_OFFER].freeze
+
+  INTERNAL_VOLUNTEER = 'internal_volunteer'.freeze
+  EXTERNAL_VOLUNTEER = 'external_volunteer'.freeze
+  VOLUNTEER_STATES = [INTERNAL_VOLUNTEER, EXTERNAL_VOLUNTEER].freeze
+
+  VOLUNTEER_ACCOUNTABLE = 'volunteer_accountable'.freeze
+  VOLUNTEER_MEMBER = 'volunteer_member'.freeze
+  VOLUNTEER_RESPONSIBLE_STATES = [VOLUNTEER_ACCOUNTABLE, VOLUNTEER_MEMBER].freeze
```

Comment:
same here, why not just an array of symbols?
That would make the later offer_state_collection etc. methods obsolete

Reply:
or even do it with enum?

Reply:
An array of symbols would also make more sense to me, unless you need to refer to the array items explicitly in the code (which doesn't seem to be the case here).

Also, it's good practice to put constants at the top of a class, after `include`'s/`extend`'s (see [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide#consistent-classes))